actors/builtin/miner/partition_state.go

```diff
@@ -345,7 +345,7 @@ func (p *Partition) TerminateSectors(store adt.Store, epoch abi.ChainEpoch, sect
 	}
 
 	powerDelta := removed.ActivePower.Add(removed.FaultyPower)
-	p.LivePower = p.LivePower.Add(powerDelta)
+	p.LivePower = p.LivePower.Sub(powerDelta)
```

Comment:
Thanks. Conventionally I have used "delta" to signify a value that has the right sign for `Add`ing to the prior. To maintain that here, please rename `powerDelta` to `powerRemoved`, which naming would have made the original bug more obvious.